packages/mobile/e2e/src/usecases/WalletConnectV1.js

```diff
@@ -149,7 +149,7 @@ export default WalletConnect = () => {
     let result = walletConnector.signTransaction(tx)
     await waitFor(element(by.text('Sign a Celo TX')))
       .toBeVisible()
-      .withTimeout(10 * 1000)
+      .withTimeout(15 * 1000)
```

Comment:
i wonder if it's worthwhile to make a wrapper for this line and the one above, like `toBeVisibleWithTimeout()` and inside that function you could control the timeout...makes it easier to maintain the timeout values?

Reply:
The Detox Maintainers have indicated they want to move away form `waitFor` syntax, but I am unsure on the priority in their roadmap: https://github.com/wix/Detox/issues/2190. Can wrap this in the utilities in another PR.

Reply:
+1 to moving it to an utils or somewhere where we can put a default timeout so it's easier to change. If they do end up changing the `waitFor` syntax we would only need to change it in one place (probably).